cranelift/filetests/filetests/wasm/load-store/make-load-store-tests.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+
+# This script generates the `heap_load_store_*.clif` test files.
```

Comment:
This comment has the wrong filename pattern in it now.

Reply:
Good catch!